app/src/parser_txdef.h

```diff
@@ -22,6 +22,8 @@ extern "C" {
 #include <stdint.h>
 #include <stddef.h>
 
+extern const char TAG_INIT[7];
```

Comment:
The array length is defined here because I wanted to remove the null terminator and simplify how the size is calculated (just using sizeof). Removing this value will imply some changes like replace sizeof by strnlen in some parts of the code. Let me check if there is any other fancy way to do this

Reply:
Yeah, I was using strlen, it's fine to do it like that I'd say 👍🏻